build.gradle

```diff
@@ -36,6 +36,16 @@ import java.nio.file.Files
 import java.nio.file.Path
 import java.security.MessageDigest
 
+plugins {
+    id 'com.gradle.build-scan' version '1.13.2'
+}
+if (properties.get("org.elasticsearch.scanTOS", "no") == "agree") {
```

Comment:
I think we should yes true/false here, not "no" and "agree".

Reply:
@rjernst done